File: 2129/go/main.go
func capitalizeTitle(title string) string {
	arr1 := strings.Split(title, " ")
	result := make([]string, 0, len(arr1))
	for _, word := range arr1 {
		if len(word) <= 2 {
			result = append(result, strings.ToLower(word))
		} else {
			result = append(result, strings.ToUpper(string(word[0]))+strings.ToLower(word[1:]))
		}
	}
	return strings.Join(result, " ")
}

File: 2129/python3/main.py
class Solution:
    def capitalizeTitle(self, title: str) -> str:
        arr = []
        for word in title.split(' '):
            if len(word) <= 2:
                arr.append(word.lower())
            else:
                arr.append(word[0].upper() + word[1:].lower())
        return ' '.join(arr)
